packages/ui/react/src/components/Forms/Input/Input.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+
+import { ErrorMessage } from '../../Primitives/ErrorMessage'
```

Comment:
Can you apply the suggestion below to use path alias instead of relative path?

```suggestion
import { ErrorMessage } from 'src/components/Primitives/ErrorMessage'
```